honeybee_grasshopper_radiance/src/HB Annual Daylight.py

```diff
@@ -66,9 +66,12 @@
             that each sensor recieves equal or more than the illuminance threshold.
             Each value is for a different sensor of the grid. These can be plugged
             into the "LB Spatial Heatmap" component along with meshes of the sensor
-            grids to visualize results. These can also be connected to the "HB
+            grids to visualize results.
+            _
+            These can also be connected to the "HB
             Spatial Daylight Autonomy" component to compute spatial daylight
-            autonomy for each grid.
+            autonomy for each grid. Note: You should ONLY do that if you have
+            a compliant IES-LM-83-12 blinds setup.
```

Comment:
I don't think this is worded correctly because sDA has a definition that exists outside of the IES-LM-83-12 recommended thresholds. And there can be many different types of sDA depending on the criteria used (eg. `sDA 200/70%`  or any other combination of thresholds). This is why the standards defines sDA separately from the thresholds and blind behavior that it recommends:

![image](https://user-images.githubusercontent.com/5567574/140191236-b04827ac-da8b-4ff0-83bb-aa464624b459.png)

If we want to make this change, I suggest changing this to something that makes it clear the values aren't **LEED compliant** unless the three criteria are met:
* threshold of 300 lux
* Percent of occupied time > 50%
* Blinds or shades operated hourly to block direct sunlight as defined in Section 2.2 of IES-LM-83-12

Reply:
Hi Chris,

I agree and disagree.
The lm83 is defining a sDA no latter if we are going leed or not. The blinds setup is a part of the sDA. If you're not following their definition, you shouldn't be calling it a sDA in my opinion. The last few lines of your screenshot shows it perfectly and they are not referring to LEED simulations specifically.

Reply:
But agree to underline that this cannot be used for LEED without blinds. We're fighting in architectural competitions where our competitors are using fake simulations, thus their simulations look more positive than ours. Their excuse? 
"Oh didn't want to purchase lm83 paper for $5, because LBT has an automatic sDA output"

Reply:
Sorry for the late response. I should have clarified that the **Definition** section of sDA in IES-LM-83-12 says clearly that there are many possible types of sDA depending on the thresholds used. And the definition of sDA exists separately from the recommended thresholds and blinds behavior.

I also should have mentioned that we need the user object exported in order to be able to mere a PR. I'll do this tomorrow unless this PR is edited.